tests/src/Kernel/BclIconPathOverrideTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel;
+
+use Drupal\Core\Theme\ThemeManagerInterface;
+use Drupal\KernelTests\KernelTestBase;
+
+/**
+ * Tests the bcl_icon_path theme override functionality.
+ */
+class BclIconPathOverrideTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'system',
+  ];
+
+  /**
+   * The theme manager service.
+   *
+   * @var \Drupal\Core\Theme\ThemeManagerInterface
+   */
+  protected ThemeManagerInterface $themeManager;
+
+  /**
+   * Set up the test environment.
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Initialize the theme manager service.
```

Comment:
We are missing the file_scan_ignore_directories behaviour as per \Drupal\Tests\oe_bootstrap_theme_helper\Kernel\BclComponentsOverrideTest::setUp().

Reply:
added